db/drivers/mysql/driver.c

```diff
@@ -16,7 +16,7 @@
 #include "globals.h"
 #include "proto.h"
 
-int db__driver_init(int argc, char *argv[])
+int db__driver_init(int argc UNUSED, char *argv[] UNUSED)
```

Comment:
I've never seen this yet, what does it mean? It's like an extra "word" (not as in a computer word) in a function signature?

I've tried looking it up, the closest I got was GNU C `__attribute__`. In that case, does it still compile correctly with clang?

Reply:
https://github.com/OSGeo/grass/blob/aaecfc8957ab906785b3231da2aed7d0861ce71a/include/grass/gis.h#L45

Reply:
Thanks! That's clear ;)